### PR TITLE
FIX 📍 "error: operator does not exist: json @> json"

### PR DESCRIPTION
**Pull request by maxlshk**
Sorry for the late update, I resolved the issue with @> operator:

image

**The problem turned out to be not in my code, but rather in the format of "skills" column in your DB.**
After few attempts of resolving the error while using `arrayContains` I kept receiving the  (SQLSTATE 42883 error), which was caused by data being in JSON and not JSONB format, which didn't support operations like `arrayContains` at all.

The only clear solution solution is to cast the JSON column to JSONB for the purpose of this operation and it turned out to be a success.

image
image

I've tested it on my local data and it seems to be working well. I used similar queries, that are used in drizzle's `arrayContains` and not `ilike`, so only full convergency of the skill will show results. If there are any issues on your end please inform me 😉

### CODE REVIEW

**Comment by maxlshk** [comment]
And yeah, I send pull request not into your main branch so that you won't need to revert changes in case there are any issues.

File: src/components/contents/ProfileGrid.tsx [hunk 1]
```diff
@@ -16,6 +23,12 @@ const ProfileGrid = async ({ page, searchParams }: Props) => {
   const currentPage = parseInt(page); // like 1
   const itemPerPage = 5; // we want to show 5 item in per pages
   const offset = (currentPage - 1) * itemPerPage; // (1 - 1) * 3 = 0
+  const searchSkills = searchParams ? searchParams.split(/[\s,]+/) : []; // regular expression to accept inputs separated by comma, space or both
+
+  // In order to get rid of the "error: operator does not exist: json @> json" I mannually cast the skills column to JSONB
+  const skillsCondition = searchSkills.length > 0
```
Comment (devlopersabbir) [inline src/components/contents/ProfileGrid.tsx]: I really appreciate your passion 🥹
One simple change is needed,
Can you make sure that, if the user types skills like `python` but our db array has `Python` then it should be searched perfectly,

Now, it does not give the result as user expects.

Can we make it whenever you write like all `uppercase` or `lowercase` or `camelcase` anything. so, it should give results perfectly

**Comment by devlopersabbir** [comment]
@maxlshk sorry for mentioning to you.
But I think you should take care of this issue that I make comments

**Comment by maxlshk** [comment]
> @maxlshk sorry for mentioning to you. But I think you should take care of this issue that I make comments

Hey, @devlopersabbir I'd gladly examine your comments, but it looks like your review is still pending:
image
Maybe you forgot to submit it, so please make sure I can see it 🤓

**Review by devlopersabbir — CHANGES_REQUESTED** [review]
Just add make changes with awesome commit message 😇
I'll be happy to marge it :)

File: src/components/contents/ProfileGrid.tsx (hunk 1 above)
Comment (maxlshk) [reply src/components/contents/ProfileGrid.tsx]: Just added a commit, that should resolve the case sensitive search issue. I'm pretty sure now that all inputs and JSON strings from db are converted to lowercase everything should work just fine ✅

**Review by devlopersabbir — APPROVED** [review]
@maxlshk thanks for your latest update 😍

**Pull request merged**